Formula/scw.rb

```diff
@@ -1,35 +1,29 @@
 class Scw < Formula
-  desc "Manage BareMetal Servers from command-line (as easily as with Docker)"
+  desc "Command-line Interface for Scaleway"
   homepage "https://github.com/scaleway/scaleway-cli"
-  url "https://github.com/scaleway/scaleway-cli/archive/v1.20.tar.gz"
-  sha256 "4c50725be7bebdab17b8ef77acd230525e773631fef4051979f8ff91c86bebf8"
-  head "https://github.com/scaleway/scaleway-cli.git"
-
-  bottle do
-    cellar :any_skip_relocation
-    sha256 "1534981c7a030117408095b219d79614333acbb4ead6ecf4b11625acf7b186c7" => :catalina
-    sha256 "1c7a69355bafd8c035bb2878f6c9c7a9a013dacc8cdbdc1997b66bc49f7fd7f0" => :mojave
-    sha256 "87a2f754ab3e1c5434a2b24b855de6e5da39990bf9cf558416fc2e259d66d0ab" => :high_sierra
-    sha256 "b5727720c3f4173012c81b8dfee1a942262d430ae6951c0f39a9c28aefc21b83" => :sierra
-  end
+  url "https://github.com/scaleway/scaleway-cli/archive/v2.0.0.tar.gz"
+  sha256 "3f219c83e3766eb253791ac74c6bf92f8b2234e147cf5d268254cf5cc6ab7d03"
 
   depends_on "go" => :build
 
   def install
-    ENV["GOPATH"] = buildpath
-    ENV["GOBIN"] = buildpath
-    (buildpath/"src/github.com/scaleway/scaleway-cli").install Dir["*"]
+    system "go", "build", *std_go_args, "./cmd/scw"
+
+    (zsh_completion/"_scw").write <<~EOS
+      eval "$(scw autocomplete script shell=zsh)"
```

Comment:
Can we just take the output of this command instead of generating completion upon calling it?

Reply:
The output of this command might change, it is easier to manage it if we eval it like this.

Reply:
I don't see what the difference is in managing. There's only the difference that generating it upon install will avoid arbitrary command execution every time someone opens their shell.

Reply:
Fixed, I've added the output of those commands in the block